src/test/java/com/github/nylle/javafixture/FixtureTest.java

```diff
@@ -588,15 +546,11 @@ void createThroughRandomConstructor() {
             assertThat(result).isInstanceOf(TestObjectWithGenericConstructor.class);
             assertThat(result.getValue()).isInstanceOf(String.class);
             assertThat(result.getInteger()).isInstanceOf(Optional.class);
-            assertThat(result.getInteger()).isPresent();
-            assertThat(result.getInteger().get()).isInstanceOf(Integer.class);
-            assertThat(result.getPrivateField()).isNull();
         }
 
     }
 
     @Test
-    @Disabled("This is a known problem")
```

Comment:
Is this working now? 🤯

Reply:
Exactly. Eventually we will construct a Throwable with just a message or an empty constructor.